loader.js

```diff
@@ -64,6 +64,7 @@ llab.paths.css_files.push('css/default.css');
 llab.paths.scripts[0] = [];
 llab.paths.scripts[0].push(llab.CONFIG_FILE_PATH);
 llab.paths.scripts[0].push("lib/jquery/dist/jquery.min.js");
+llab.paths.scripts[0].push("//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML");
```

Comment:
lol, well the config file doesn't support custom JS...

--
Michael Ball | michaelballphoto.com

On Mon, Jul 27, 2015 at 12:17 AM, Nate Titterton <notifications@github.com>
wrote:

> In loader.js
> <https://github.com/beautyjoy/llab/pull/149#discussion_r35510287>:
>
> > @@ -64,6 +64,7 @@ llab.paths.css_files.push('css/default.css');
> >  llab.paths.scripts[0] = [];
> >  llab.paths.scripts[0].push(llab.CONFIG_FILE_PATH);
> >  llab.paths.scripts[0].push("lib/jquery/dist/jquery.min.js");
> > +llab.paths.scripts[0].push("//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML");
>
> This should absolutely be going into the bjc config file, yo
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/beautyjoy/llab/pull/149/files#r35510287>.
>